spec/requests/app_spec.rb

```diff
@@ -21,4 +21,15 @@
       expect(parsed_response).to eq(data_object_for(verse))
     end
   end
+
+  describe 'GET /devotionals' do
+    let!(:devotionals) { create_list(:devotional, 6) }
+
+    it 'returns a list of devotionals' do
+      get '/devotionals', {}, headers
+
+      expect(last_response.status).to eq(200)
+      expect(parsed_response).to eq(data_object_for(devotionals))
```

Comment:
Maybe we should change this helper to work for both multiple and single objects instead of having a different one for each, because this won't work.

Reply:
oh the data_object_for? That's where the error is coming from

Reply:
Yep, let's update that to check whether it's getting an array and then it can call `data_objects_for_multiple`. I've hit this error too and it's stupid.